test/ConverterTest.java

```diff
@@ -0,0 +1,31 @@
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+/**
+ * Created by ken-linux2 on 25/07/16.
+ */
+public class ConverterTest {
+
+   private Converter classUnderTest;
+    final String STRING_TEST="Google,2831944540,186307692,,,,GOOGLE_WOMENS_ACCESSORIES_PLA,WOMENS_ACCESSORIES,GOOGLE,PLA,CENTER_CORE,,WOMENS_ACCESSORIES,BRANDS,BRANDS,,";
```

Comment:
Probably should move inside the test as you'll want to add additional cases in the future.
For example what happens if the line is empty? What happens if is has the wrong number of columns? etc